### PR TITLE
Show pending stake in account page

### PR DESCRIPTION
**Pull request by kimpers**
image

### CODE REVIEW

File: ts/components/ui/panel_header.tsx [hunk 1]
```diff
@@ -68,8 +65,8 @@ const IconWrap = styled.div`
     border-radius: 50%;
     border: 1px solid ${colors.border};
     position: absolute;
-    right: 0;
-    bottom: 0;
+    right: 9px;
+    bottom: 9px;
```
Comment (kimpers) [inline ts/components/ui/panel_header.tsx:69]: Adjusted these slightly to make it look more like in the design

File: ts/hooks/use_stake.ts [hunk 2]
```diff
@@ -53,8 +53,9 @@ export const useStake = (networkId: ChainId, providerState: ProviderState): UseS
         const _contractAddresses = getContractAddressesForChainOrThrow(networkId);
 
         setOwnerAddress(_ownerAddress);
+        // NOTE: staking proxy has state and is a delegate proxy to staking contract, it can be used to initialize both contracts
         setStakingContract(
-            new StakingContract(_contractAddresses.staking, providerState.provider, {
+            new StakingContract(_contractAddresses.stakingProxy, providerState.provider, {
```
Comment (kimpers) [inline ts/hooks/use_stake.ts:58]: This was correct after all 😄

Comment (johnrjj) [reply ts/hooks/use_stake.ts:58]: haha no worries!

**Review by johnrjj — APPROVED** [review]
Nice! LGTM

**Pull request merged**